package.json

```diff
@@ -16,8 +17,12 @@
   "author": "",
   "license": "BSD",
   "dependencies": {
+    "@aws-sdk/client-s3": "^3.208.0",
+    "@aws-sdk/url-parser": "^3.208.0",
     "@types/bcrypt": "^5.0.0",
-    "@types/sha.js": "^2.4.0"
+    "@types/sha.js": "^2.4.0",
+    "lodash": "^4.17.21",
```

Comment:
We don't want to add lodash to anywhere new, in fact we're gradually phasing it out as most of its features can now be handled natively by javascript
```suggestion
```

Reply:
Sorry for the noise, and thanks for the detailed comments; this version had some leftovers from previous tests.